packages/shared-ini-file-loader/src/loadSharedConfigFiles.ts

```diff
@@ -20,6 +20,12 @@ export interface SharedConfigInit {
    * `~/.aws/config` otherwise.
    */
   configFilepath?: string;
+
+  /**
+   * Configuration files are normally cached after the first time they are loaded. When this
+   * property is set, the provider will always reload any configuration files loaded before.
+   */
+  ignoreCache?: boolean;
```

Comment:
drawing on existing precedent, maybe `forceRefresh` makes sense: https://github.com/aws/aws-sdk-js-v3/blob/a79f962873ff11a3d6f98199224155bd2ddc0b1e/packages/property-provider/src/memoize.ts#L70

btw, is there an existing `refresh()` or `reload()` function somewhere in this area? (I didn't look much)

Reply:
I don't think an existing function exists. I kind of expected that, but no.